### PR TITLE
Update non-experimental queries with standardized metadata.

### PR DESCRIPTION
**Pull request by NateD-MSFT**
Standardizes metadata for all non-experimental queries to match our requirements.  One experimental query also got standardized.

This does **not** update the messages output by said queries to match internal autofiling requirements.

## Checklist for Pull Requests

- [X] Description is filled out.
- [ ] Only one query or related query group is in this pull request.
- [ ] The version number on changed queries has been increased via the `@version` comment in the file header.
- [X] All unit tests have been run: ([Test README.md](src\drivers\test\README.md)).
- [X] Commands `codeql database create` and `codeql database analyze` have completed successfully.
- [ ] A .qhelp file has been added for any new queries or updated if changes have been made to an existing query.

### CODE REVIEW

File: src/drivers/general/queries/DefaultPoolTag/DefaultPoolTag.ql
Comment (lauragonzalezzz) [inline src/drivers/general/queries/DefaultPoolTag/DefaultPoolTag.ql:1]: Looks like this is missing SECURITY SEVERITY and IMPACT. See https://www.osgwiki.com/wiki/CodeQL_Query_Development_%26_Testing#Query_Metadata for the full table of requirements.

OPAQUE ID and SCOPE are not in the table, where are these requirements coming from?
If they are new requirements, please update the osgwiki table.

File: src/drivers/general/queries/ExtendedDeprecatedApis/ExtendedDeprecatedApis.ql [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
  * @feature.area Multiple
  * @repro.text The following code locations contain calls to an unsafe, deprecated function or macro.
- * @kind problem
- * @id cpp/windows/drivers/queries/extended-deprecated-apis
+ * @owner.email: sdat@microsoft.com
+ * @opaqueid C28719
  * @problem.severity warning
  * @precision high
  * @tags correctness
+ *       security
```
Comment (lauragonzalezzz) [inline src/drivers/general/queries/ExtendedDeprecatedApis/ExtendedDeprecatedApis.ql:18]: Accidental tab?

File: src/drivers/general/queries/IrqlNotSaved/IrqlNotSaved.ql [hunk 2]
```diff
@@ -10,11 +11,12 @@
  * @impact Insecure Coding Practice
  * @repro.text This function has a parameter annotated \_IRQL\_saves\_, but does not have the system IRQL saved to it.
  * @owner.email sdat@microsoft.com
- * @kind problem
+ * @opaqueid C28158
  * @problem.severity warning
  * @precision medium
  * @tags correctness
  *       wddst
```
Comment (lauragonzalezzz) [inline src/drivers/general/queries/IrqlNotSaved/IrqlNotSaved.ql:18]: Extra tab?

File: src/drivers/general/queries/IrqlNotSaved/IrqlNotSaved.ql [hunk 3]
```diff
@@ -59,7 +61,7 @@ class IrqlFlowConfiguration extends DataFlow::Configuration {
 }
 
 /**
- * Represents a "fundamental" function that restores IRQL, i.e. one defined
+ * A "fundamental" function that restores IRQL, i.e. one defined
```
Comment (lauragonzalezzz) [inline src/drivers/general/queries/IrqlNotSaved/IrqlNotSaved.ql:64]: What do the quotes around the word fundamental mean?

File: src/drivers/general/queries/DefaultPoolTag/DefaultPoolTag.ql
Comment (NateD-MSFT) [reply src/drivers/general/queries/DefaultPoolTag/DefaultPoolTag.ql:1]: Yep, they appear to be new standards from the 1ES/ECA standards - see the link to "CodeQL Query-Writing Standards | 1ES On EngHub" in internal deliverable 44049070.  I'll update our OSGWiki page.

File: src/drivers/general/queries/ExtendedDeprecatedApis/ExtendedDeprecatedApis.ql (hunk 1 above)
Comment (NateD-MSFT) [reply src/drivers/general/queries/ExtendedDeprecatedApis/ExtendedDeprecatedApis.ql:18]: Intentional spacing to match SDL style.

File: src/drivers/general/queries/IrqlNotSaved/IrqlNotSaved.ql (hunk 2 above)
Comment (NateD-MSFT) [reply src/drivers/general/queries/IrqlNotSaved/IrqlNotSaved.ql:18]: At "wddst"? In that case, it's intentional spacing to match SDL style.

File: src/drivers/general/queries/IrqlNotSaved/IrqlNotSaved.ql (hunk 3 above)
Comment (NateD-MSFT) [reply src/drivers/general/queries/IrqlNotSaved/IrqlNotSaved.ql:64]: I used the quotes here to indicate that I was using 'fundamental' in a slightly unique way that I defined in the next clause, and that this was not a standardized definition of 'fundamental'.  But it's probably a little confusing.  I'll see if I can rewrite.

File: src/drivers/general/queries/DefaultPoolTag/DefaultPoolTag.ql
Comment (lauragonzalezzz) [reply src/drivers/general/queries/DefaultPoolTag/DefaultPoolTag.ql:1]: When I authored the query metadata doc, I included the requirements for both ECA and CodeQL internal, so please double check that all the requirements have been fulfilled and update the docs accordingly so I can approve the PR.

Comment (NateD-MSFT) [reply src/drivers/general/queries/DefaultPoolTag/DefaultPoolTag.ql:1]: I did go ahead and update the query metadata table in OSGWiki with the fields asked for by the current requirements, along with links to the current standards.

**Comment by NateD-MSFT** [comment]
@lauragonzalezzz I updated the remaining queries with the standardized metadata and regenerated the associated SARIF files.  Please take a quick look (you shouldn't need to do an exhaustive review of each file.)

**Review by lauragonzalezzz — APPROVED** [review]

**Pull request merged**